contributing.md

```diff
@@ -10,21 +10,43 @@ You can always add to an API, you cannot ever remove anything from one. If the d
 
 That's why many of the guidelines of this document are obvious and serve only one purpose: Simplicity.
 
-## A good pull request
-Every contribution has to come with:
+ - [Questions](#question)
+ - [Issues or Bugs](#issue)
+ - [Submitting a pull request](#pr)
+ - [Quality insurance for pull requests for XAML controls](#xaml)
+ - [General rules](#rules)
+ - [Naming conventions](#naming)
+ - [Documentation](#documentation)
+ - [Files and folders](#files)
+
+
+## <a name="question"></a> Questions
+Please do not open issues for general support questions and keep our GitHub issues for bug reports and feature requests. There is a much better chance of getting your question answered on [StackOverflow](https://stackoverflow.com/search?q=uwp+community+toolkit) where questions should be tagged with the tag `uwp-community-toolkit`
+
+## <a name="issue"></a> Found a Bug?
+If you find a bug, you can help us by
+[submitting an issue](https://github.com/Microsoft/UWPCommunityToolkit/issues). Even better, you can
+[submit a Pull Request](#pr) with a fix.
+
+## <a name="pr"></a> Submitting a pull request
+For every controbution, you must:
+
+* test your code with the [supported SDKs](readme.md#supported)
+* follow the [quality guidance](#xaml), [general rules](#rules) and [naming convention](#naming)
+* target master branch (or an appropriate rel branch if appropriate for a bug fix)
```

Comment:
should `rel branch` be abbreviated? I would think `release branch` would be better here

Reply:
Done